package.json

```diff
@@ -26,7 +26,7 @@
   "devDependencies": {
     "@playwright/test": "1.38.0",
     "@prettier/plugin-xml": "^3.1.0",
-    "@types/node": "^20.6.0",
+    "@types/node": "18.6.5",
```

Comment:
Use the correct version of `@types/node` for the node version specified in `pom.xml`